bundles/org.openhab.binding.goecharger/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -38,6 +41,10 @@
 				<description>The IP address of the Go-eCharger</description>
 				<context>network-address</context>
 			</parameter>
+			<parameter name="apiVersion" type="integer" required="true" min="1" max="2">
```

Comment:
Documented as optional since it should default to 1. Please check if it should be required here.